### PR TITLE
Added a thousand separator function

### PR DESCRIPTION
**Pull request by DJkal11**

### CODE REVIEW

**Review by adlerjohn — COMMENTED** [review]
Thank you for the contribution! Revert changes to `package-lock.json` please.

File: scripts/fuel-tx.ts [hunk 1]
```diff
@@ -28,11 +35,11 @@ async function run() {
...
   const from = "0x0101010101010101010101010101010101010101010101010101010101010101";
   const to = genBytes32();
-  const amount = BigNumber.from(1);
+  const amount = thousandSeparator(BigNumber.from(1));
   await provider.sendTransaction({
     type: TransactionType.Script,
-    gasPrice: BigNumber.from(0),
-    gasLimit: BigNumber.from(1000000),
+    gasPrice: thousandSeparator(BigNumber.from(0)),
+    gasLimit: thousandSeparator(BigNumber.from(1000000)),
```
Comment (adlerjohn) [inline scripts/fuel-tx.ts:42]: This doesn't quite seem right? You shouldn't be adding separators when _sending_ a tx, that should just be the number. It's only _displaying_ the numbers that should involve separators.

Comment (DJkal11) [reply scripts/fuel-tx.ts:42]: oh i see, which page displays the numbers.

Comment (luizstacio) [reply scripts/fuel-tx.ts:42]: You can use something like this: https://github.com/FuelLabs/swayswap-demo/blob/415af2b3ce6bc49242741ac0d7e66efe59bdecc6/client/src/components/Assets.tsx#L80

Comment (DJkal11) [reply scripts/fuel-tx.ts:42]: Cool, I Think I'll use this, Thank you.

File: scripts/fuel-tx.ts [hunk 2]
```diff
@@ -4,6 +4,7 @@ import { hexlify } from "@ethersproject/bytes";
 
 const genBytes32 = () => hexlify(new Uint8Array(32).map(() => Math.floor(Math.random() * 256)));
 
+
```
Comment (adlerjohn) [inline scripts/fuel-tx.ts:7]: Remove

File: src/pages/TransactionPage/index.tsx [hunk 3]
```diff
@@ -40,11 +40,20 @@ import { ExpandIcon, ShrinkIcon } from "../../components/Icons";
 import { trimAddress } from "../../utils";
 import { InputFragment, OutputFragment, useTransactionPageQuery } from "./__generated__/operations";
 
+
```
Comment (adlerjohn) [inline src/pages/TransactionPage/index.tsx:43]: Remove

File: src/pages/TransactionPage/index.tsx [hunk 4]
```diff
@@ -254,7 +263,7 @@ function UTXOOutputBox({ output, expanded }: { output: OutputFragment; expanded:
                     {trimAddress(output.to)}
                   </UTXODetailsLink>
                 );
-              }
+              }
```
Comment (adlerjohn) [inline src/pages/TransactionPage/index.tsx:266]:
```suggestion
              }
```

Comment (DJkal11) [reply src/pages/TransactionPage/index.tsx:265]: I fixed the first two conflicts but this last one confuses me.

Comment (adlerjohn) [reply src/pages/TransactionPage/index.tsx:265]: It looks like you added an extra whitespace here. Suggestion removes it.

File: src/pages/TransactionPage/index.tsx [hunk 5]
```diff
@@ -45,6 +45,14 @@ export function TransactionPage() {
   const { data } = useTransactionPageQuery({ variables: { id: transaction } });
   const tx = data?.transaction;
 
+  function thousandSeparator(number : any) {
```
Comment (QuinnLee) [inline src/pages/TransactionPage/index.tsx:48]: Both `tx.gasPrice` and `tx.gasLimit` are `string`, i think this can be type `string`

**Comment by luizstacio** [comment]
@DJkal11 are you still working on this?

**Comment by DJkal11** [comment]
@luizstacio Yep, Im still working on this.

File: src/pages/TransactionPage/index.tsx (hunk 5 above)
Comment (DJkal11) [reply src/pages/TransactionPage/index.tsx:48]: Thank you, will try type string.

File: src/pages/TransactionPage/index.tsx (hunk 4 above)
Comment (DJkal11) [reply src/pages/TransactionPage/index.tsx:265]: I see, thank you.

**Comment by luizstacio** [comment]
This was fixed on #61. Thank you for the contribution @DJkal11

**Pull request closed without merging**